OpenRA.Game/Network/OrderManager.cs

```diff
@@ -216,12 +216,14 @@ public class ChatLine
 		public readonly Color Color;
 		public readonly string Name;
 		public readonly string Text;
+		public readonly Color TextColor;
 
-		public ChatLine(Color c, string n, string t)
+		public ChatLine(Color c, string n, string t, Color textColor)
```

Comment:
It would be good if we could make the naming and ordering a bit more obvious, maybe something like `public ChatLine(string name, Color nameColor, string text, Color textColor)` -- and then carry this back through the rest of the plumbing.